C/test.c

```diff
@@ -528,6 +528,12 @@ static void regression_tests(void) {
     diff = (double)(end - start) / CLOCKS_PER_SEC;
     bound = (double)(sizeof_regression3) * secondsPerWU;
     printf("cpu_time_used by regression3: %f s.  (Should be less than %f s.)\n", diff, bound);
+    if (diff <= bound) {
+      successes++;
+    } else {
+      failures++;
+      printf("regression3 took too long.\n");
+    }
```

Comment:
dd9a973a1a1eecdbe76573423c242d5ae0f15f1e: The outcome of this check depends on the machine it is run on. Is this OK for our CI? If regression3 takes too long, then this seems like a softer error than if, say, the cmr was falsely computed.

Reply:
I'm torn on the issue because making the test based on time means it is inherently a flaky test.  On the other hand, I do want to know if it ever fails and not just ignore the issue.

Reply:
Hm, yes, having a flaky test may be annoying for users and distributors, too. But I don't have a great suggestion to improve it. (You could count CPU cycles of just the test thread, but this is just less flaky and probably much less portable.)

Reply:
I've opened issue #202.